types/floatthead/index.d.ts

```diff
@@ -0,0 +1,96 @@
+// Type definitions for floatthead 2.0
+// Project: https://github.com/mkoryak/floatThead
+// Definitions by: Soner Köksal <https://github.com/renjfk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jquery" />
+
+type TableFunction = ($table: JQuery) => void;
```

Comment:
Since these are globals right now, could you put these in a `FloatThead` namespace?